frontend/src/component/common/Limit/Limit.tsx

```diff
@@ -127,7 +127,9 @@ export const Limit: FC<{
                 <LimitExplanation>
                     <Link
                         target='_blank'
-                        to={'https://docs.getunleash.io/reference/limits'}
+                        to={
```

Comment:
we can also drop {} since it's a string literal

Reply:
Ah, yeah, good point. Done in afa00294426d01b82c1a2e2b177cb2b2e744fa9c